### DISCUSSION TITLE
Is project dead?

### DISCUSSION THREAD

**Discussion by vladmyr** [roc-streaming/roc-toolkit#154] [Discuss]
I was watching the repo for quite some time and it is pity to see no progress starting from January. Are there any plans on development continuation?

**Comment by gavv** [comment]
Thanks for asking.

I'm currently a bit overworked at my job. Roc will be the top priority when I will be able to organize my free time better. I can't give a concrete estimate now, but I'm working on it :) Actually, there are quite a lot of plans limited only with the presence of spare time.

**Comment by gavv** [comment]
The first version of API and documentation is ready. We'll fix a couple of critical bugs and make a release in a few weeks, I think.

**Comment by gavv** [comment]
We've finally released 0.1.0 today! There is still a long way to go thought :)
